examples/server-example.py

```diff
@@ -74,11 +72,26 @@ def multiply(parent, x, y):
     uri = "http://examples.freeopcua.github.io"
     idx = server.register_namespace(uri)
 
-    # get Objects node, this is where we should put our custom stuff
+
+    # create a new node type we can instantiate in our address space
+    base_type = server.get_root_node().get_child(["0:Types", "0:ObjectTypes", "0:BaseObjectType"])
```

Comment:
Good idea. Maybe the same as the other shortcuts server.get_ base_type_
node() ?

On Tue, Jul 19, 2016, 17:50 Andrew <notifications@github.com> wrote:

> In examples/server-example.py
> <https://github.com/FreeOpcUa/python-opcua/pull/244#discussion_r71366632>:
>
> > @@ -74,11 +72,26 @@ def multiply(parent, x, y):
> >      uri = "http://examples.freeopcua.github.io"
> >      idx = server.register_namespace(uri)
> >
> > -    # get Objects node, this is where we should put our custom stuff
> > +
> > +    # create a new node type we can instantiate in our address space
> > +    base_type = server.get_root_node().get_child(["0:Types", "0:ObjectTypes", "0:BaseObjectType"])
>
> Thoughts on making a shortcut to this? server.BASETYPE.add_object_type()
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/FreeOpcUa/python-opcua/pull/244/files/88f2ae0f075c2248c3421c956ab06051cd3d1ec9..785f185239fb050ac90cca77296cff6d996c58ee#r71366632>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACcfzq2VjBxiqtNbGOP1bWZLnrW8UuRJks5qXPI3gaJpZM4JLHjh>
> .
>